alacritty.yml

```diff
@@ -63,6 +63,13 @@
   #     - buttonless: Title bar, transparent background and no title bar buttons
   #decorations: full
 
+  # Background opacity
+  #
+  # Window opacity as a floating point number from `0.0` to `1.0`.
+  # The value `0.0` is completely transparent and `1.0` is opaque.
+  #opacity: 1.0
```

Comment:
Do you think we should leave this as `opacity` or change it to `transparency`? I personally think it's not worth it to change it since most Alacritty users will likely be used to the existing behavior already and inverting it will just cause unnecessary confusion. But one could argue that it's more consistent with the new `transparent_background_colors` option.